src/Error/ExceptionTrap.php

```diff
@@ -169,6 +169,8 @@ public function register(): void
         set_exception_handler($this->handleException(...));
         register_shutdown_function($this->handleShutdown(...));
         static::$registeredTrap = $this;
+
+        ini_set('assert.exception', '1');
```

Comment:
Do we want this on for 'production' mode too? Or would it be better to have the `assert()` compiled away by using `-1` in non-debug mode?

I think having the assert() compiled away could help improve baseline performance as we can have the assertions excluded from opcodes and not included during execution.

Reply:
Ok. I missed the part in the docs where code generation can only be controlled by the initial file. That makes sense considering what it does. I can update the deployment docs to include changing that config value.